hunts/views.py

```diff
@@ -162,8 +162,8 @@ def post(self, request, pk):
             if google_api_client:
                 sheet = google_api_client.create_google_sheets(name)
             else:
-                # TODO(erwa): This should incur a warning.
```

Comment:
Did @erwa mean a warning log or a user-visible warning on the site? If the latter, we should probably leave the TODO. Log still seems useful though.

Reply:
Not sure; personally I'm fine with no user warning (if smallboard is being run without google credentials it's going to be obvious anyway since there won't be any sheets created) but open to other opinions